tsconfig.json

```diff
@@ -3,13 +3,14 @@
     "declaration": true,
     "esModuleInterop": true,
     "importHelpers": true,
+    "lib": ["es2020"],
     "module": "commonjs",
+    "noImplicitAny": true,
     "outDir": "lib",
     "rootDir": "src",
     "strict": true,
-    "target": "es2020",
-    // Remove `dom` once there's an acceptable fix for https://github.com/axios/axios/issues/3219
-    "lib": ["dom", "es2020"]
+    "strictNullChecks": true,
+    "target": "es2020"
```

Comment:
^

Reply:
Can I use `es2020` for the `lib` and target `es2019`?

Reply:
Updated to `es2019`